client/views/omnichannel/directory/chats/contextualBar/VoipInfo/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { ReactElement } from 'react';
+
+import { useRoom } from '../../../../../room/contexts/RoomContext';
+import { VoipInfo } from './VoipInfo';
+
+const VoipInfoWithData = ({ tabBar: { close } }: any): ReactElement => {
+	const room = useRoom();
+	console.log(room);
+	const { servedBy } = room;
+
+	const guest = {};
```

Comment:
I think all the data you need for the guest can be acquired from an IVoipRoom I did this on a pretty similar component 

`	const { servedBy, queue, v, fname, callDuration, callTotalHoldTime, callEndedAt, callWaitingTime } = room; `

Reply:
For sure, this is not ready... I did the fix earlier... 🤣